beacon-chain/forkchoice/doubly-linked-tree/unrealized_justification_test.go

```diff
@@ -0,0 +1,186 @@
+package doublylinkedtree
+
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/config/params"
+	types "github.com/prysmaticlabs/prysm/consensus-types/primitives"
+	"github.com/prysmaticlabs/prysm/testing/require"
+	"golang.org/x/net/context"
```

Comment:
why use a different library? 

```suggestion
	"context"
```

Reply:
I have no clue, this is vim-go, thanks for catching this